library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -601,6 +601,13 @@ libspdm_return_t libspdm_send_spdm_request(libspdm_context_t *spdm_context,
         }
     }
 
+    if ((spdm_context->connection_info.capability.max_spdm_msg_size != 0) &&
+        (request_size > spdm_context->connection_info.capability.max_spdm_msg_size)) {
+        LIBSPDM_DEBUG((LIBSPDM_DEBUG_ERROR, "request_size > rsp max_spdm_msg_size\n"));
+        return LIBSPDM_STATUS_PEER_BUFFER_TOO_SMALL;
+    }
+    LIBSPDM_ASSERT (request_size <= spdm_context->local_context.capability.max_spdm_msg_size);
```

Comment:
Sorry, this should probably also include `max_spdm_msg_size != 0` in the assertion.

Reply:
This is local_context.
I dont think we need add zero check.